### PR TITLE
Add signing service inspection commands

### PR DESCRIPTION
**Pull request by mdellweg**
fixes #TBA

### CODE REVIEW

**Comment by mdellweg** [comment]
I want to find a way to make this testable before merging.

File: pulpcore/cli/core/context.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def upload(self, file: IO[bytes], chunk_size: int = 1000000, check_exists: bool
 
 
 class PulpExporterContext(PulpEntityContext):
-    ENTITY = "PulpExporter"
+    ENTITY = "pulp exporter"
```
Comment (ggainey) [inline pulpcore/cli/core/context.py:89]: Is this supposed to be part of this PR?

File: pulpcore/cli/core/context.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ class PulpExporterContext(PulpEntityContext):
 
 
 class PulpExportContext(PulpEntityContext):
-    ENTITY = "PulpExport"
+    ENTITY = "pulp export"
```
Comment (ggainey) [inline pulpcore/cli/core/context.py:99]: Same question here

File: pulpcore/cli/core/context.py [hunk 3]
```diff
@@ -267,7 +267,7 @@ def scope(self) -> Dict[str, Any]:
 
 
 class PulpImporterContext(PulpEntityContext):
-    ENTITY = "PulpImporter"
+    ENTITY = "pulp importer"
```
Comment (ggainey) [inline pulpcore/cli/core/context.py:270]: ...and here :)

File: tests/assets/setup_signing_service.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    with open(public_key_file, "rb") as key:
+        public_key = key.read()
+
+    AptReleaseSigningService.objects.create(
+        name="sign_deb_release",
+        script=script_path,
+        public_key=public_key,
+        pubkey_fingerprint="6EDF301256480B9B801EBA3D05A5E6DA269D9D98",
+    )
```
Comment (ggainey) [inline tests/assets/setup_signing_service.py:46]: :+1:

File: tests/assets/sign_deb_release.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  --output "${INLINE_SIGNATURE_PATH}" \
+  --local-user "${GPG_KEY_ID}" \
+  "${RELEASE_FILE}"
+
+echo "{" \
+       \"signatures\": "{" \
+         \"inline\": \""${INLINE_SIGNATURE_PATH}"\", \
+         \"detached\": \""${DETACHED_SIGNATURE_PATH}"\" \
+       "}" \
+     "}"
```
Comment (ggainey) [inline tests/assets/sign_deb_release.sh:30]: :+1:

**Review by ggainey — CHANGES_REQUESTED** [review]
Can we detach the import/export changes from this PR? Otherwise, this looks great!

File: pulpcore/cli/core/context.py (hunk 3 above)
Comment (mdellweg) [reply pulpcore/cli/core/context.py:270]: sure

**Review by ggainey — APPROVED** [review]
Looks good!

**Pull request merged**